cms/db/contest.py

```diff
@@ -79,7 +79,7 @@ class Contest(Base):
     languages = Column(
         ARRAY(String),
         nullable=False,
-        default=["C11 / gcc", "C++11 / g++", "Pascal / fpc"])
+        default=["C11 / gcc", "C++11 / g++", "Pascal / fpc", "C++17 / g++"])
```

Comment:
Maybe it would make more sense to replace C++11 with C++17, what do you think?

It might make sense to allow both in some contests, but a more sensible default probably is to allow only one.

Reply:
I think you are right. Especially since some contestants do not know the difference between the c++11 and the c++17 standards and will usually choose the later one.

Reply:
I changed the default setting for g++ from C++11 to C++17.